modules/home-manager/yazi.nix

```
@@ -14,5 +14,6 @@ in
 
   config = lib.mkIf enable {
```

```suggestion
  config = lib.mkIf enable {
    assertions = [
      (lib.ctp.assertXdgEnabled "yazi")
    ];
```
oops, might have been a bit too eager 😆 
an assertion will need to be used here to make sure `xdg` is enabled

---

good catch, forgot that you could disable xdg